ramls/time-period.json

```diff
@@ -18,11 +18,25 @@
         "Months"
       ],
       "default" : "Days"
+    },
+    "closeLibraryDateManagement": {
+      "type": "string",
+      "description": "enum for closeLibraryDateManagement",
+      "enum":[
+        "Keep_the_current_due_date",
+        "Move_to_the_end_of_the_previous_open_day",
+        "Move_to_the_end_of_the_next_open_day",
+        "Keep_the_current_due_date_time",
+        "Move_to_end_of_current_service_point_hours",
+        "Move_to_beginning_of_next_open_service_point_hours"
+      ],
+      "default" : "Keep_the_current_due_date"
     }
```

Comment:
I don't think this property should be added to the time period structure. 

The time period structure is intended to be a general representation of time periods and closed library due date management is not an inherent property of a time period. Rather it is a separate concept that might be used to change operations involving a time period.

The description of this change suggests the intention is to associate the closed due date management to a service period. That suggests to me that it should be included in the service point representation not the time period.

Reply:
Hi @marcjohnson-kint , the intention of putting closeLibraryDateManagement in timeperiod structure , as to include it in holdShelfExpiryPeriod object ( I see the mention raml is time-period.json)  , which eevntually will be part of Service point object .

Reply:
I understand that intention. I am suggesting that it could be better to include the property in the service point rather than the time period.

From a domain modelling perspective, this structure is intended to represent a period of time. A period of time is completely unaware of the context in which might be used. 

In this case, part of that context is, that a closed library due date management policy is to be applied to hold shelf expiry at this service point. That is a characteristic of the hold shelf expiry beyond the basic period of time.

Does that make sense?

Reply:
We already have examples of [similar modelling|https://github.com/folio-org/mod-circulation-storage/blob/174ccdd8b5c684e4774d2ffc1c8a4ac1f8f38a2f/ramls/loan-policy.json#L25-L33] in loan policies.